src/demo/routes/routes.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+import PrivateRoute from "./guards/PrivateRoute";
+import HomeScreen from "../pages/Home";
+import Callback from "../pages/Callback";
+import MainLayout from "../layout/MainLayout";
+import mainRoutes from "./mainRoutes";
+import AuthProvider from "../providers/AuthProvider";
+
+export default function RootRoutes(): JSX.Element {
+  return (
+    <Router>
+      <Routes>
+        <Route path="/" element={<HomeScreen />} />
```

Comment:
if the user tries to go to the "/" home page and are already logged in, will they get redirected to the /user-home page?

Reply:
is this correct?